packages/node/package.json

```diff
@@ -36,6 +36,7 @@
     "@vercel/node-bridge": "3.0.0",
```

Comment:
Can we move `@vercel/ncc` back to devDependencies?

Reply:
Good call. Moved: [`8e4c7fe` (#8032)](https://github.com/vercel/vercel/pull/8032/commits/8e4c7fecf4ab3ccdc104d0239899f5994f7f7c10)